extra/cockpit/foreman-cockpit.service

```diff
@@ -1,14 +1,12 @@
 [Unit]
 Description=Foreman Cockpit
-Documentation=https://theforeman.org
+Documentation=Websocket authentication service for Foreman-Cockpit integration
 After=network.target remote-fs.target nss-lookup.target
 
 [Service]
-Environment=XDG_CONFIG_DIRS=/etc/foreman/
-Environment=FOREMAN_COCKPIT_SETTINGS=/etc/foreman/cockpit/foreman-cockpit-session.yml
-Environment=ADDRESS=127.0.0.1
-Environment=PORT=9999
-ExecStart=/usr/libexec/cockpit-ws --no-tls --address $ADDRESS --port $PORT
+EnvironmentFile=/etc/foreman/foreman-cockpit.conf
```

Comment:
Packaging change needed here - this file must be shipped otherwise the unit fails to start.